docs/resources/job-device_compat.csv

```diff
@@ -43,7 +43,17 @@ AWS      ,BRAKET_OQC_LUCY                   ,✓      ,             ,✓
 AWS      ,BRAKET_QUERA_AQUILA               ,✓      ,             ,✓          ,
 AWS      ,BRAKET_RIGETTI_ASPEN_M_3          ,✓      ,             ,✓          ,
 Google   ,CIRQ_LOCAL_SIMULATOR              ,✓      ,✓            ,✓          ,✓
-Google   ,PROCESSOR_RAINBOW                 ,✓      ,             ,           ,
-Google   ,PROCESSOR_WEBER                   ,✓      ,             ,           ,
-Google   ,IONQ_SIMULATOR                    ,✓      ,             ,           ,
-Google   ,IONQ_QPU                          ,✓      ,             ,           ,
\ No newline at end of file
+Google   ,PROCESSOR_RAINBOW                 ,✓      ,             ,✓          , 
+Google   ,PROCESSOR_WEBER                   ,✓      ,             ,✓          , 
+Google   ,IONQ_SIMULATOR                    ,✓      ,             ,           , 
+Google   ,IONQ_QPU                          ,✓      ,             ,           ,
+Azure    ,IONQ_SIMULATOR                    ,✓      ,             ,           ,
+Azure    ,IONQ_QPU                          ,✓      ,             ,           ,
+Azure    ,IONQ_QPU_ARIA_1                   ,✓      ,             ,           ,
+Azure    ,IONQ_QPU_ARIA_2                   ,✓      ,             ,           ,
+Azure    ,QUANTUM_SIM_H1_1                  ,✓      ,             ,           ,
+Azure    ,QUANTUM_SIM_H1_1SC                ,✓      ,             ,           ,
+Azure    ,QUANTUM_SIM_H1_1E                 ,✓      ,             ,           ,
+Azure    ,RIGETTI_SIM_QVM                   ,✓      ,             ,           ,
```

Comment:
were all these devices tested ?

Reply:
done

Reply:
the tests should come under the long flag (or maybe an expensive flag x) )

Reply:
long flag should be ok